packages/twenty-server/src/engine/metadata-modules/remote-server/remote-server.service.ts

```diff
@@ -99,6 +104,98 @@ export class RemoteServerService<T extends RemoteServerType> {
     );
   }
 
+  async updateOneRemoteServer(
```

Comment:
This logic looks like almost a duplicate of `createOneRemoteServer`. It may be factorized into commun functions for each step. You just need to ensure first that this is still similar even if you start updating only the fields that are set as asked above.